howdoi/howdoi.py

```diff
@@ -203,13 +231,20 @@ def _get_questions(links):
     return [link for link in links if _is_question(link)]
 
 
+
```

Comment:
No need for an extra line here

Reply:
k

Reply:
6abb8420a67cf0a65abc91fcd5b9462735aaa286